README.rst

```diff
@@ -33,16 +31,31 @@ Supervised Manual Ghost Update
 current version to the latest is supported. Always ensure that 
 you have a tested backup before proceeding with software updates.
 
-Assuming that updating to the latest Ghost version is supported. 
-update from the command line::
+Update NodeJS (recommended but perhaps required; example updating to latest
```

Comment:
> recommended but perhaps required ...

The `but` in this sentence is a little confusing

> recommended **and** perhaps required ...

might be better

Reply:
Good call @OnGle Although as @qq7 is busy, I'll take care of this...